### ISSUE TITLE
4.9.0 not published on rubygems.org

### ISSUE THREAD

**Issue by southwolf** [thoughtbot/factory_bot_rails#250]
4.9.0 of this gem was released on Oct 24, but has not been published on rubygems.org.

When I switch from `factory_girls_rails`, I have to downgrade the version number, which is wierd.

**Comment by joshuaclayton** [comment]
Hi @southwolf! 4.9.0 is the version we published factory_girl_rails (and factory_girl) under, as it introduces the deprecation warning encouraging developers to update their codebase to factory_bot_rails and factory_bot (both of which were released as version 4.8.2).

If your Gemfile currently has a dependency on factory_bot_rails/factory_bot and you're not getting any deprecation warnings, excellent! You should expect to be on 4.8.2 for that version, as it's the latest version available for the gem.

If you're getting deprecation warnings from factory_girl/factory_bot, you'll want to update your Gemfile and update all occurrences of `FactoryGirl` to `FactoryBot`.

Best of luck!

**Comment by southwolf** [comment]
Hi @joshuaclayton , I used `factory_girls_rails v4.9.0`, and got the notice to move to `factory_bot_rails`. Since they are the same ones with just different names, I assume it's reasonable to change `girl` to `bot`, and keep the same version number. 

But `factory_bot_rails` has no `4.9.0` on `rubygems.org`, I had to change `4.9.0` down to `4.8.2` to make it work. 

Does it sound like a stepping back instead of forward?
